src/org/armedbear/lisp/Lisp.java

```diff
@@ -2286,6 +2492,136 @@ public LispObject execute(LispObject key, LispObject value)
       }
     };
 
+  // ### %set-stepping-task-on
```

Comment:
A more "Lisp-y" way might be to implement `defsetf-expanders` on various specials, but since these reader/writers are essentially direct toggles of Java booleans, I guess this is fine.